setup-scripts/log-checker.py

```diff
@@ -40,13 +41,27 @@ async def on_ready():
 async def run_checks():
     print("Running Skynet portal log checks")
     try:
+        await check_load_average()
         await check_docker_logs()
 
     except: # catch all exceptions
         trace = traceback.format_exc()
         await send_msg(client, "```\n{}\n```".format(trace), force_notify=False)
 
 
+# check_load_average monitors the system's load average value and issues a
+# warning message if it exceeds 10.
+async def check_load_average():
+    uptime_string = os.popen("uptime").read().strip()
+    # pattern = ""
+    if sys.platform == "Darwin":
+        pattern = "^.*load averages: \d*\.\d* \d*\.\d* (\d*\.\d*)$"
+    else:
+        pattern = "^.*load average: \d*\.\d*, \d*\.\d*, (\d*\.\d*)$"
```

Comment:
what are those numbers exactly?

Reply:
The average number of processes either using CPU or waiting for it over the last 1, 5, and 15 minutes.

Good article: http://www.brendangregg.com/blog/2017-08-08/linux-load-averages.html
Also Wikipedia: https://en.wikipedia.org/wiki/Load_(computing)

Reply:
Can we indicate that in the message?

Reply:
Can we add some context to those in the message @ro-tex ?
Maybe `1.73 (1 min), 0.60 (5 min), 7.98 (15 min)` or sth like that?

Reply:
That would be quite redundant. It's a little like adding context to a time string: `duration: 1:34:00` -> `duration: 1 (hour) 34 (minutes) 0 (seconds)`.

Reply:
Seems better to report the output of `uptime` verbatim, however I would make it clear in the message that this string comes from `uptime`.

Reply:
I wasn't aware of the `uptime` syntax up until just now compared to a common thing like a clock.

Reply:
You are right, it isn't the same thing. It requires some arcane knowledge to understand what it means and it also requires some quite specific knowledge about the system in order to assess whether it's a problem, e.g. a load average of 5 on a single core system is a bad thing but it's perfectly fine on a quad-core hyper-threaded CPU.

I think I'm with Marcin on this - providing the `uptime` output verbatim should be fine. I also agree on the note that we need to make it clear that this is coming from that specific command. I'll make those changes.